sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -2003,6 +2027,85 @@ def set_premium_page_blob_tier(self, premium_page_blob_tier, **kwargs):
         except StorageErrorException as error:
             process_storage_error(error)
 
+    def _set_blob_tags_options(self, blob_tags=None, **kwargs):
+        # type: (Optional[Dict[str, str]], **Any) -> Dict[str, Any]
+        headers = kwargs.pop('headers', {})
+
+        tags = serialize_blob_tags(blob_tags)
+
+        options = {
+            'tags': tags,
+            'version_id': kwargs.pop('version_id', None),
+            'timeout': kwargs.pop('timeout', None),
+            'cls': return_response_headers,
+            'headers': headers}
+        options.update(kwargs)
+        return options
+
+    @distributed_trace
+    def set_blob_tags(self, blob_tags=None, **kwargs):
+        # type: (Optional[Dict[str, str]], **Any) -> Dict[str, Any]
+        """The Set Tags operation enables users to set tags on a blob or specific blob version, but not snapshot.
+
+        :param blob_tags:
+            Blob tags
+        :type blob_tags: dict(str, str)
+        :keyword str version_id:
+            The version id parameter is an opaque DateTime
+            value that, when present, specifies the version of the blob to delete.
+            It for service version 2019-10-10 and newer.
+        :keyword bool validate_content:
```

Comment:
I'm curious how validate content is applicable for this function? Does changing the tags affect the md5?
We should definitely change the docstring description - which talks about data transfer and no content is transferred in this function.

Reply:
It's to validate the tags, since tags are sent as body